client/trino-jdbc/pom.xml

```diff
@@ -91,6 +86,12 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>io.opentelemetry.instrumentation</groupId>
+            <artifactId>opentelemetry-okhttp-3.0</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
```

Comment:
will code fail if not provided. Or just not expose any observability metrics?

Reply:
No, there is a code that tries to apply an OTEL and fallback to non-traced execution :)